examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet382.java

```diff
@@ -159,35 +189,40 @@ public void handleEvent(Event e)  {
 						drawImages(mainGC, gcData, "Disabled",220, disabledImageWithDataProvider);
 						drawImages(mainGC, gcData, "Greyed",260, greyImageWithDataProvider);
 
-						mainGC.drawText("--Image with ImageData--", 20, 290);
-						drawImages(mainGC, gcData, "Normal",320, imageWithData);
-						drawImages(mainGC, gcData, "Disabled",360, disabledImageWithData);
-						drawImages(mainGC, gcData, "Greyed",400, greyImageWithData);
-
-						mainGC.drawText("--ImageDataProvider Copy--", 20, 430);
-						drawImages(mainGC, gcData, "Normal",460, imageWithDataCopy.get());
-						drawImages(mainGC, gcData, "Disabled",500, disabledImageWithDataCopy.get());
-						drawImages(mainGC, gcData, "Greyed",540, greyImageWithDataCopy.get());
-
-						mainGC.drawText("--ImageGcDrawer--", 20, 570);
-						drawImages(mainGC, gcData, "Normal", 600, imageWithGcDrawer);
-						drawImages(mainGC, gcData, "Disabled", 640, disabledImageWithGcDrawer);
-						drawImages(mainGC, gcData, "Greyed", 680, greyImageWithGcDrawer);
-
-						mainGC.drawText("--Transparent ImageGcDrawer--", 20, 710);
-						drawImages(mainGC, gcData, "Normal", 740, imageWithTransparentGcDrawer);
-						drawImages(mainGC, gcData, "Disabled", 780, disabledImageWithTransparentGcDrawer);
-						drawImages(mainGC, gcData, "Greyed", 820, greyImageWithTransparentGcDrawer);
-
-						mainGC.drawText("--Image with PlainImageData--", 20, 850);
-						drawImages(mainGC, gcData, "Normal", 880, imageWithPlainImageData);
-						drawImages(mainGC, gcData, "Disabled", 920, disabledImageWithPlainImageData);
-						drawImages(mainGC, gcData, "Greyed", 960, greyImageWithPlainImageData);
-
-						mainGC.drawText("--Image with PlainImageData (Handle Created Before Grey or Disabled Image Created)--", 20, 1000);
-						drawImages(mainGC, gcData, "Normal", 1040, imageWithPlainImageDataWithHandleAlreadyCreated);
-						drawImages(mainGC, gcData, "Disabled", 1080, disabledImageWithHandleCreated);
-						drawImages(mainGC, gcData, "Greyed", 1120, greyImageWithHandleCreated);
+						mainGC.drawText("--ImageDataAtSizeProvider--", 20, 290);
+						drawImagesAtSize(mainGC, gcData, "Normal",320, imageWithDataAtSizeProvider);
```

Comment:
Since this and the following lines are touched anyway, maybe we can auto-format them (there are whitespaces missing)?